monero-outreach-docs/translations/pt-br/storyboard/blockchain_gender_pt_br.md

```diff
@@ -0,0 +1,78 @@
+# Disparidade de gênero no blockchain: Problema ou potencial?
+
+Monica Mizzi | 12/09/18 Uma entrevista com @sunchakr da Monero Talk.
+
+Discussões e debates sobre blockchain são frequentemente voltados para a análise de aspectos técnicos dessa tecnologia revolucionária, mas **existe um tópico não técnico que sempre chama a atenção: mulheres no blockchain.**
+
+Esse tópico tem provocado aplausos e vaias tanto de mulheres quanto de homens. Embora seja difícil negar a disparidade de gênero nessa indústria, o que [não carece de estatísticas](https://cointelegraph.com/news/women-in-blockchain-and-crypto-how-to-tackle-gender-inequality) que destaquem o menor número de mulheres investindo e trabalhando na indústria, as opiniões das pessoas sobre o assunto ainda são muito divergentes.
+
+De um lado estão aqueles que acreditam que abordar diretamente a disparidade de gênero no blockchain - e na indústria de tecnologia em geral - são essenciais para ajudar a alavancar o status e a participação das mulheres nessa indústria. Do outro lado, estão aqueles que não acreditam que se quer exista um problema de representação.

```

Comment:
Remove "nessa indústria". Example: "alavancar o status e a participação das mulheres."

Reply:
Remove the comma here: "Do outro lado estão aqueles que não acreditam"